packages/core/src/App/Containers/Routes/routes.jsx

```diff
@@ -34,21 +45,71 @@ class Routes extends React.Component {
         this.props.setAppRouterHistory(this.props.history);
     }
 
+    componentDidUpdate(pp, ps) {
```

Comment:
Do you mean previous prop and state here? If so, I think complete variables will be more clear. Perhaps `previous_prop` & `previous_state`? Just a suggestion 😄

Reply:
Done